exo-19-json.js

```diff
@@ -0,0 +1,99 @@
+var JOURNAL = [
```

Comment:
Where is the exercise to convert this array into JSON object?

Où est l'exercice pour convertir ce tableau en objet JSON?